AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/DefaultKeyStorageInterface.dfy

```diff
@@ -324,32 +349,29 @@ module DefaultKeyStorageInterface {
                     //#  - Item: A [record formatted item](#record-format) constructed from the version input
                     //#  - ConditionExpression: `attribute_not_exists(branch-key-id)`
                     //#  - TableName: the configured Table Name
-                    CreateTransactWritePutItem(
+                    TransactCreateHKey(
```

Comment:
Most of our code uses the name branch key. Starting to use HKey seems like an antipattern

Reply:
The Storage change introduced `EncryptedHierarchicalKey`.  
I abbreviated this to HKey.  
I can expand it to `EncryptedHierarchicalKey`.  
But I want to use what ever lexicon the Storage layer establishes.